.travis.yml

```diff
@@ -50,8 +50,8 @@ matrix:
         # value so the tests pass. The real environment variable will be checked during deployment.
         - docker-compose run -e SITE_URL=https://treeherder.dev -e TREEHERDER_DEBUG=False backend python -bb ./manage.py check --deploy --fail-level WARNING
         # Using `-bb` mode to surface BytesWarnings: https://docs.python.org/3.7/using/cmdline.html#cmdoption-b
-        - docker-compose run backend python -bb -m pytest tests/ --runslow --ignore=tests/selenium/
-
+        - docker-compose run backend bash ./tests/coverage.sh
+     
```

Comment:
I see that you used `bash` and I misled you.  Good job for working around my mistakes!

Reply:
@klahnakoski thank you! If you can provide me the token id of the repo, I can make the changes